docs/claim-transparency.md

```diff
@@ -0,0 +1,339 @@
+# Claim Transparency
+
+The following diagram shows the journey that the software takes from code to an application used by
+the end user. During this process several artifacts are generated and transformed to another. The
+premise of software supply chain security is that many things could go wrong during this process,
+either due to human error or attacks on the software supply chain by malicious actors.
+
+![The journey of a software binary](docs/images/journey.png)
+
+To provide assurances to the end users about the security and privacy of the software they interact
+with, in the transparent release project our goal is to provide transparency into this process. In
+our solution, in each step of the process the artifacts are being reviewed and audited and the
+result, a claim about the security and privacy properties of the software artifact, is signed and
+published into a transparency log. The goal is to make these claims easily discoverable so that
+they can be used in the assessment of privacy policies.
+
+The following diagram shows what needs to be reviewed or audited in each step of the software
+development process, from coding until the software is shipped.
+
+![The claim transparency architecture](docs/images/design.png)
+
+## The Claim Format
+
+We propose the following structure, based on the [in-toto Statement standard](https://github.com/in-toto/attestation/blob/main/spec/README.md#statement),
+for specifying security and privacy claims. This format is meant to be generic and allow specifying
+different types of claims.
+
+```json
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "subject": [{ ... }],
+
+  "predicateType": "https://github.com/project-oak/transparent-release/schema/claim/v1",
+  "predicate": {
+    "issuer": {
+      "id": "<URI>"
+    },
+    "claimType": "<URI>",
+    "metadata": {
+      "issuedOn": "<TIMESTAMP>",
+      "validFrom": "<TIMESTAMP>",
+      "expiresOn": "<TIMESTAMP>",
+    },
+    "claimSpec": { /* object */ },
+    "evidence": [
+      {
+        "role": "<STRING>",
+        "uri": "<URI>",
+        "digest": { /* DigestSet */ }
+      }
+    ]
+  }
+}
+```
+
+### Fields
+
+This section describes the semantics of each field in the claim format:
+
+- **subject** _(array of objects, required)_:
+  Set of artifacts that the claim applies to. Usually only one artifact (e.g., source code, or some binary).
```

Comment:
The in-toto format allows a list, in our work we often are only interested in one subject. I don't see the contradiction. Our usage is a specialisation.

Reply:
Does it make sense that we say "For Transparent Release we have usually only one artifact" ?

Reply:
For transparent release, is it usually? I thought it should be always. Unless we change things in the future. Otherwise, what is the semantics if multiple subjects are listed? Does the claim apply to all of them?

Reply:
I think the current wording is fine.

Reply:
This is the generic format, and does not intend to get into the specific semantics of a single use case. Removed "usually".